### PR TITLE
feat(keycloak-operator): Update Keycloak to 25.0.0 and update CRDs to match

### PR DESCRIPTION
**Pull request by maxthier**
# Description
This update Keycloak to 25.0 and updates the CRD's and values to match the new version.

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog with an `artifacthub.io/changes` annotation in `Chart.yaml`, check the [example](docs/development.md#Changelog) in the documentation.
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Comment by hairmare** [comment]
i'll wait until we land #1277 before pushing this

File: charts/keycloak-operator/values.yaml
Comment (vmaillot) [inline charts/keycloak-operator/values.yaml:95]: nice : D

File: charts/keycloak-operator/values.yaml
Comment (vmaillot) [inline charts/keycloak-operator/values.yaml:111]:
```suggestion
    host: ~
```

**Review by vmaillot — APPROVED** [review]

**Pull request merged**
